src/backend/dx12/src/device.rs

```diff
@@ -2867,16 +2867,14 @@ impl d::Device<B> for Device {
         }
     }
 
-    unsafe fn get_fence_status(&self, _fence: &r::Fence) -> Result<bool, d::DeviceLost> {
-        unimplemented!()
+    unsafe fn get_fence_status(&self, fence: &r::Fence) -> Result<bool, d::DeviceLost> {
+        Ok(fence.borrow().raw.GetCompletedValue() != 0)
```

Comment:
why `borrow()`?

Reply:
Would you mind adding the additional check against `UINT64_MAX` for device removal?